Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -27,6 +27,11 @@ class BaseHTTPClientTests: TestCase {
     override func setUpWithError() throws {
         try super.setUpWithError()
 
+        #if os(watchOS)
+        // See https://github.com/AliSoftware/OHHTTPStubs/issues/287
+        try XCTSkipIf(true, "OHHTTPStubs does not currently support watchOS")
+        #endif
```

Comment:
would love to replace OHHTTPStubs altogether some day

Reply:
Yup..